### PR TITLE
feat: add smartbch chain

### PR DESCRIPTION
**Pull request by Gauravk0029**

### CODE REVIEW

File: chains/smartbch/logo.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="230" height="230">
```
Comment (JaunePomme) [inline chains/smartbch/logo.svg:2]: Not sure you need these width and height

Can minimify probably the logo with https://jakearchibald.github.io/svgomg/

**Comment by JaunePomme** [comment]
I would say add the changeset (minor) and run the yarn prettier as well just in case, looks ok overall imo

**Comment by Fazleiw** [comment]
can you provide test relay proof

File: chains/smartbch/logo.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+</svg>
```
Comment (paulbalaji) [inline chains/smartbch/logo.svg:5]: cropped+minified
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="20 10.6 188.6 208.7"><path fill="#74DD52" d="m176 15 3 1c13 7 22 17 27 31 4 12 2 24-4 36l-10 13 1 1 5 7 2 2c8 11 10 24 8 37-2 9-6 15-12 22l-2 2c-8 9-20 15-33 16-7 1-13 4-19 8l-13 7a171826 171826 0 0 0-13 7l-3 2-10 5-3 1c-8 5-15 6-24 6h-3c-15 0-27-5-38-16a50 50 0 0 1-14-36c1-14 7-24 16-34l-1-1c-10-9-15-19-16-32s4-25 13-35l2-2c8-9 19-14 30-15 10-2 17-6 25-11l13-7 18-10c16-10 37-13 55-5Zm-34 30a4016 4016 0 0 1-28 15l11 7 2 1a2065 2065 0 0 0 14 8l1 1c7 3 15 5 22 2 6-2 10-6 13-12 2-6 1-11-2-16-4-5-10-10-16-10-7 0-11 1-17 4ZM55 84c-4 6-6 10-5 17l5 8 2 3c6 4 12 5 19 5 7-2 12-6 18-9a653 653 0 0 1 11-5l1-1c4-2 5-3 7-6l-2-1-9-5-4-2-3-2-3-1-10-6c-10-5-20-2-27 5Zm80 37-6 4-2 1-11 6v2a6462 6462 0 0 0 22 12l2 1c7 4 13 7 22 6 7-2 11-6 15-12 2-5 2-10 0-16-5-7-10-10-18-12-9 0-16 4-24 8Zm-79 36c-4 5-6 10-5 16 2 6 6 10 11 14 6 3 12 3 18 1l21-11 3-1 10-7-28-16-2-1c-10-5-21-3-28 5Z"/><path fill="#C5E34F" d="m176 15 3 1c13 7 22 17 27 31 4 12 2 24-4 36l-11 14-3-2-4-2-2-1c-13-7-27-9-41-6-8 2-15 6-21 9l-9 4v-3l-11-6-3-2c-14-9-14-9-30-10-6 2-11 5-14 11-2 5-3 9-1 14s5 9 11 11 11 2 18 0l4-1 2-1c-6 6-12 8-20 9-11 2-20 7-29 13-11-9-16-20-18-34-1-13 4-25 13-35l2-2c8-9 19-14 30-15 10-2 17-6 25-11l13-7 18-10c16-10 37-13 55-5Zm-34 30a4016 4016 0 0 1-28 15l11 7 2 1a2065 2065 0 0 0 14 8l1 1c7 3 15 5 22 2 6-2 10-6 13-12 2-6 1-11-2-16-4-5-10-10-16-10-7 0-11 1-17 4Z"/></svg>
```

File: chains/smartbch/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://smartscout.cash/api
+    family: blockscout
+    name: smartbchscan
```
Comment (paulbalaji) [inline chains/smartbch/metadata.yaml:5]:
```suggestion
    name: SmartBCH Explorer
```

File: chains/smartbch/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+name: smartbch
+nativeToken:
+  decimals: 18
+  name: BCH
+  symbol: BCH
+protocol: ethereum
+rpcUrls:
+  - http: https://smartbch.greyh.at
+  - http: https://rpc.smartbch.org
+  - http: https://global.uat.cash
```
Comment (paulbalaji) [inline chains/smartbch/metadata.yaml:27]: third rpc doesn't work
```suggestion
rpcUrls:
  - http: https://smartbch.greyh.at
  - http: https://rpc.smartbch.org
```

**Review by paulbalaji — COMMENTED** [review]
almost there

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**